docs/source/interfaces/Engineering_Diffraction.rst

```diff
@@ -258,6 +260,62 @@ This tab uses the algorithms :ref:`Rebin <algm-Rebin>` and :ref:`Rebin
 <algm-RebinByPulseTimes>` to bin the data in different ways when
 converting event data into histogram data.
 
+Fitting
+-------
+
+.. warning:: This is a new capability that is currently in a very
+             early stage of definition and implementation. Not all
+             options may be supported and/or consistent at the moment.
+
+The Fitting tab provides a graphical interface which fits an expected
+diffraction pattern and visualises them. The pastern is specified by
+providing a list of dSpacing values where Bragg peaks are expected.
+The algorithm :ref:`EnggFitPeaks<algm-EnggFitPeaks>` used in the
+background fit peaks in those areas using a peak fitting function.
+
+To use the Fitting tab, user is required to provide a focused workspace
+as Focus Run input, list of expected peaks which can be either by browsing
+a (*CSV*) file or entering within the text-field and simply click on the Fit
+button.
+
+Parameters
+^^^^^^^^^^
+
+These parameters are required to process Fitting successfully:
+
+Focused Run #:
+  Focused workspace directory or selected with the help of browse button.
+  Focused workspace can be generated with the help of
+  :ref:`focus-Engineering_Diffraction-ref` tab, the output folder
+  directory can be set in the :ref:`setting-Engineering_Diffraction-ref`
+  tab under the *Focusing settings* section.
+
+Peaks:
+  A list of dSpacing values to be translated into TOF to find expected
+  peaks. These peaks can be manually written or imported by selected a
+  (*CSV*) file.
+
+Output
+^^^^^^
+
+Once the Fit button has been clicked, wait until the Fitting process has
+completed and upon completion you should be able to view on the Fitting
+tab plots the focused workspace in the background in black, whereas the
+expected peaks plotted in various colours over lapping the focused
+workspace peaks.
+
+The interface will also generate workspaces that can be inspected in the
+workspaces window:
+
+1. The *engggui_fitting_fitpeaks_param* Table workspace
+   with the parameters of the peaks found and fitted.
+2. The *engggui_fitting_focused_ws* Focused workspace also loaded
+   so the fitted data can be compared with focused data
+3. The *engggui_fitting_single_peaks* workspace within each workspace
+   index representing individual expected peak.
+
+.. _setting-Engineering_Diffraction-ref:
+
```

Comment:
Very good documentation. With this already in place we won't need to change much, just tiny updates and additions in the future as we narrow down more details.